app.go

```diff
@@ -115,10 +128,7 @@ func (app *App) AddHandler(spec Spec) error {
 		route.Headers(spec.Headers...)
 	}
 
-	// vulcan registration
-	if app.registry != nil && spec.Register != false {
-		app.registerLocation(spec.Methods, spec.Path, spec.Scopes, spec.Middlewares)
-	}
+	app.registerLocation(spec.Methods, spec.Path, spec.Scopes, spec.Middlewares)
```

Comment:
With Vulcand, do you turn off registration completely or only for some handlers? You can still choose to disable registration entirely. This does away with per-handler registration because I haven't actually seen it used anywhere.